plugins/vst_base/RemoteVstPlugin.cpp

```diff
@@ -812,6 +909,46 @@ void RemoteVstPlugin::initEditor()
 	// 64-bit versions of Windows use 32-bit handles for interoperability
 	m_windowID = (intptr_t) m_window;
 #endif
+
```

Comment:
The above `#endif` is a wrapper for Linux/Window build. Since this is now part of a `#ifndef NATIVE_LINUX_VST` section is that needed anymore? Assuming `NATIVE_LINUX_VST` already means we have `LMMS_BUILD_LINUX` defined.

Reply:
Yes - we support Windows VSTs on Linux using Wine, and the remote plugin host for these is built with `LMMS_BUILD_LINUX` defined (since it's for a Linux system), but not `NATIVE_LINUX_VST` (since it's for hosting Windows VSTs). The different behaviour is needed so that on Windows we get an `HWND`, but on Linux we get Wine's underlying X11 window handle.